README.md

```diff
@@ -53,3 +53,15 @@ box. We added support for additional fingerprint and bad header sources in the c
 you want to have it periodically update the fingerprint and bad header files it uses to analyze traffic.
 
 The intended entrypoint to the mitmengine package is through the `Processor.Check` function, which takes a user agent and client request fingerprint, and returns a mitm detection report. Additional API functions will be added in the future to allow for adding new signatures to a running process, for example.
+
+## How to Contribute
+
+As browser and middleware fingerprints quickly become outdated, we are actively seeking to update the fingerprint repository with new samples. To contribute fingerprint samples, please submit a PR with the following:
+- Pcap (generated by `tcpdump` or `wireshark`, for example) containing a Client Hello generated by the TLS client to fingerprint at `testdata/pcaps/<desc>/handshake.pcap`, where `<desc>` is a unique and descriptive name for the pcap sample
```

Comment:
Should this only cover the Client Hello message (display filter `ssl.handshake.type==1`) or cover a full stream (`tcp.stream==1234`)? What about preceding DNS packets and such?

Should this describe the required steps for those who are not familiar with Wireshark? It might not be a good idea to include LAN traffic in the capture for example.

Reply:
Only `ssl.handshake.type == 1` is necessary since we just need the Client Hello message. I'll add some more instructions for actually collecting the pcaps!

Reply:
I updated the README with a section on generating both TLS and HTTP header samples.